lib/routes/verify.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
   validate: {
     payload: {
       token: validators.token.required(),
-      email: Joi.boolean().default(true)
+      email: Joi.boolean().default(false)
```

Comment:
@rfk hm so current version of the PR will stop sending the email unless `email:true` is specified, might break something?

Reply:
Right.  Unfortunately I don't see a good way around this.  Any reliers that still expect the email here *should* be explicitly requesting it rather than relying on the default value here, but how can we check that without breaking any that aren't?  I think they've all been updated but it's hard to be sure.

Perhaps we should aim to do this as a point-release so we can carefully QA it (perhaps with that other breaking OAuth change, if we don't manage to confirm it from request logs?)